lib/ui/page/home/page/chat/widget/chat_item.dart

```diff
@@ -245,8 +246,13 @@ class ChatItemWidget extends StatefulWidget {
           onTap: isLocal
               ? null
               : () {
-                  final Iterable<GalleryAttachment> attachments =
-                      onGallery?.call() ?? media;
+                  if (onGallery == null) {
+                    // [onTap] still needs to be invoked to ensure [ContextMenu]
+                    // doesn't get closed, when this is being built within it.
+                    return;
+                  }
```

Comment:
I think `null`ifying the `onGallery` callback is a bit cleaner than adding separate flag for preventing the `GalleryPopup` from being opened, what's yours thoughts?

Reply:
@SleepySquash Yes, I think that would be better approach than `disableGallery` flag